### ISSUE TITLE
systemd[1]: Failed to populate /etc with preset unit settings, ignoring: Cannot assign requested address.

### ISSUE THREAD

**Issue by enihcam** [systemd/systemd#7100]
### Submission type

 - Bug report

### systemd version the issue has been seen with

> 235

### Used distribution

> Archlinux

### In case of bug report: Expected behaviour you didn't see

> no failure of populating /etc

### In case of bug report: Unexpected behaviour you saw

> systemd[1]: Failed to populate /etc with preset unit settings, ignoring: Cannot assign requested address.

### In case of bug report: Steps to reproduce the problem

> update systemd to v235 and reboot

**Comment by djgera** [comment]
An important thing here and missing from report, such message appears only at initramfs stage.

**Comment by poettering** [comment]
I figure we should never execute the preset logic in the initrd. That said, it should be a NOP anyway in the initrd...

Any chance you can rerun this with "systemd.log_level=debug" on the kernel cmdline? Do you see anything interesting regarding unit file enablement in the logs before the line above in the logs if you do so?

**Comment by Vincent43** [comment]
In my case this line disappeared after setting systemd.log_level=debug in cmdline.

**Comment by loqs** [comment]
To add to the previous  comment with the addition of systemd.log_level=debug on this system
both of the following messages are absent
```
Running with unpopulated /etc.
Failed to populate /etc with preset unit settings, ignoring: Cannot assign requested address
```
without systemd.log_level=debug
```
systemd 235 running in system mode. (+PAM -AUDIT -SELINUX -IMA -APPARMOR +SMACK -SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD -IDN2 +IDN default-hierarchy=hybrid)
Detected architecture x86-64.
Running in initial RAM disk.
Running with unpopulated /etc.
No hostname configured.
Set hostname to <archlinux>.
Initializing machine ID from random generator.
Failed to populate /etc with preset unit settings, ignoring: Cannot assign requested address
Started Dispatch Password Requests to Console Directory Watch.
Reached target Timers.
Journal started
```
with systemd.log_level=debug
```
systemd 235 running in system mode. (+PAM -AUDIT -SELINUX -IMA -APPARMOR +SMACK -SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD -IDN2 +IDN default-hierarchy=hybrid)
Found entry what=/dev/disk/by-uuid/6923081d-3c75-4675-86c6-02a1a856e4e1 where=/sysroot type=n/a
Creating /run/systemd/generator/systemd-fsck-root.service
Parsing /etc/fstab
Parsing /sysroot/etc/fstab
Journal started
```

**Comment by vheikkin** [comment]
I saw the same log after upgrading to v235 when there was a init script installed /etc/init.d (to be exact it was for libfuse). The outcome was that no presets were configured during the first boot so some services were in incorrect state. For me, this had nothing to do with initramfs.

After removing the init script (and a new install), everything was working again as expected.

**Comment by poettering** [comment]
Fix waiting in #7360
